trace-dispatcher/src/Cardano/Logging/Tracer/EKG.hs

```diff
@@ -84,3 +89,9 @@ ekgTracer storeOrServer = liftIO $ do
                         Right server -> creator2 name server
             let rgsMap' = Map.insert name gauge rgsMap
             pure (rgsMap', gauge)
+
```

Comment:
Could you provide in a comment what the implemented output format is? Like what to expect when presenting a `[("key1", "value1"), ("key2", "value2")]` for example?

Reply:
❤️